### ISSUE TITLE
Get field mapping with defaults doesn't populate `fielddata` and `norms` fully

### ISSUE THREAD

**Issue by clintongormley** [elastic/elasticsearch#12388]
```
PUT my_index/my_type/1
{
   "foo": "bar"
}

GET _mapping/field/foo?include_defaults
```

returns:

```
              "foo": {
                 "type": "string",
                 "boost": 1,
                 "index": "analyzed",
                 "store": false,
                 "doc_values": false,
                 "term_vector": "no",
                 "norms": {
                    "enabled": true
                 },
                 "index_options": "positions",
                 "analyzer": "default",
                 "similarity": "default",
                 "fielddata": {},
                 "null_value": null,
                 "include_in_all": false,
                 "position_offset_gap": 0,
                 "search_quote_analyzer": "default",
                 "ignore_above": -1
              }
```

It doesn't include:
- `norms.loading`
- `fielddata.loading`
- `fielddata.format`

**Comment by clintongormley** [comment]
Related to #12394

**Comment by clintongormley** [comment]
Fixed in 5.0
